blockmanager.go

```diff
@@ -791,7 +792,7 @@ func (b *blockManager) checkBlockForHiddenVotes(block *dcrutil.Block) {
 	// calculated.
 	template.Block.ClearSTransactions()
 	updatedTxTreeStake := make([]*dcrutil.Tx, 0,
-		votesTotal+len(oldTickets)+len(oldRevocations))
+		len(newVotes)+len(oldTickets)+len(oldRevocations))
```

Comment:
Any reason why you're calling `len(newVotes)` when `votesTotal` is available?

Reply:
matches the range